_docs-sources/iac/stay-up-to-date/updating.md

```diff
@@ -29,6 +35,97 @@ module "my_instance" {
 ```
 
 After making the change, run `terraform plan`, inspect the output to ensure it looks as you expect, then run `terraform apply`.
+</TabItem>
+<TabItem value="Terragrunt" label="Terragrunt">
+
+Below is a module block referencing version `0.15.3` of the `single-server` submodule from the `terraform-aws-server` module.
+
+To update to version `0.15.4`, you update the value to the right of `ref=` in the source attribute. Since the version number denotes that this update is backwards compatible, it should not require any other changes.
+
+```hcl
+terraform {
+  # Old
+  # source = "github.com:gruntwork-io/terraform-aws-server.git//modules/single-server?ref=v0.15.3"
+  # New
+  source = "git::git@github.com:gruntwork-io/terraform-aws-server.git//modules/single-server?ref=v0.15.4"
+}
+
+generate "provider" {
+  path = "provider.tf"
+  if_exists = "overwrite_terragrunt"
+  contents = <<EOF
+provider "aws" {
+  region = "us-west-2"
+}
+EOF
+}
+
+inputs = {
+  name = "my_instance"
+  ami = "ami-99999999999999999"
+  instance_type = "t2.medium"
+  keypair_name = ""
+  vpc_id = "vpc-1234567890123456"
+  subnet_id = "subnet-23456789012345678"
+  attach_eip = false
+}
+```
+
+After making the change, run `terragrunt plan`, inspect the output to ensure it looks as you expect, then run `terragrunt apply`.
+
+</TabItem>
+<TabItem value="Terragrunt with _envcommon" label="_envcommon (Terragrunt)">
+
+Below is an example using the `_envcommon` pattern to reference version `0.15.3` of the `single-server` submodule from the `terraform-aws-server` module.
+
+To update to version `0.15.4`, you update the value to the right of `ref=` in the source attribute. Since the version number denotes that this update is backwards compatible, it should not require any other changes. When following the `_envcommon` pattern, there are two places that reference the git tag created by the release — the `.hcl` file with the reference to the module in the `_envcommon` directory, and the environment and region specific reference to that file.
```

Comment:
```suggestion
To update to version `0.15.4`, you update the value to the right of `ref=` in the source attribute. Since the version number denotes that this update is backwards compatible, it should not require any other changes. When following the `_envcommon` pattern, there are two places that reference the git tag created by the release — the `.hcl` file with the reference to the module in the `_envcommon` directory, and the environment and region specific reference to this _envcommon file.
```

Reply:
Not sure if this actually makes the sentence more clear or not, but thought I'd try it out. Feel free to reject.

Reply:
Good call - I updated to `specific references to the _envcommon file`

Reply:
Much more clear - thanks!